requirements.txt

```diff
@@ -1,7 +1,8 @@
 setuptools>=18.8.1
```

Comment:
this could probably mostly (or entirely) be replaced with `-e .` to install sops from `setup.py`...